test/CLI.js

```diff
@@ -810,6 +810,17 @@ module.exports = (testRunner, jsonFixtures, csvFixtures) => {
     });
   });
 
+  testRunner.add('should support flattening JSON with nested arrays using the flatten transform, when arrays a longer in non-0 index entries', (t) => {
```

Comment:
The same test should be added to the other test suites (JSON2CSVParser, JSON2CSVAsyncParser, JSON2CSVTransform) to ensure that it works in all of them

Reply:
Ok, only just say this repo so I'll try figure this out, cheers